crates/cast/src/lib.rs

```diff
@@ -350,6 +350,26 @@ pub fn print_formatted(output: Vec<(&str, String)>, json: bool, error: bool) ->
     Ok(())
 }
 
+fn json_value_to_string(value: &Value, value_format: ValueFormat) -> Option<String> {
```

Comment:
Are we sure there's no some handling for it built-in in serde?

Reply:
This one has a somewhat unique behaviour

Reply:
What do you mean by `unique behaviour`, it is not captured by the name. Please rename it to be accurate, current name suggest it just dumps json.

Reply:
Also could we unit test it?

Reply:
[239a777](https://github.com/foundry-rs/starknet-foundry/pull/1067/commits/239a777c680437e95eb5102e11c70a05f116e823)